rio_cogeo/utils.py

```diff
@@ -0,0 +1,17 @@
+"""rio_cogeo.utils: Utility functions."""
+
+import rasterio
+
+
+def get_overview_level(src_path, minsize=512):
+    """Calculate the maximum overview level."""
+    with rasterio.open(src_path) as src:
+        width = src.width
+        height = src.height
+    nlevel = 0
+    overview = 1
```

Comment:
```suggestion
    overview = 1

```

A little space here too.